src/ui/public/vislib/visualizations/point_series/area_chart.js

```diff
@@ -96,7 +96,8 @@ export default function AreaChartFactory(Private) {
 
       function y1(d) {
         const y0 = d.y0 || 0;
-        return yScale(y0 + d.y);
```

Comment:
this was actually causing errors even before, but with derivative it became more obvious as derivative would always return empty value for the first bucket (or first n buckets in case of (n+1)-th derivative)